packages/cds-odata-annotation-converter/test/__mocks__/vscode.js

```diff
@@ -0,0 +1,76 @@
+const Uri = {
```

Comment:
We shouldn't have any dependencies on VSCode here and hence this should not be needed.